.travis.yml

```diff
@@ -1,20 +1,33 @@
 language: c
 
 env:
-  - AFL_I_DONT_CARE_ABOUT_MISSING_CRASHES=1 AFL_NO_UI=1
+  - AFL_I_DONT_CARE_ABOUT_MISSING_CRASHES=1 AFL_NO_UI=1 AFL_STOP_MANUALLY=1
+  - AFL_I_DONT_CARE_ABOUT_MISSING_CRASHES=1 AFL_NO_UI=1 AFL_EXIT_WHEN_DONE=1 
+  # This config is disabled because the fuzz target tested does not crash
```

Comment:
Comment should be # TODO: test AFL_BENCH_UNTIL_CRASH once we have a target that crashes